### DISCUSSION TITLE
Why `send_task` does not return Response of sqs.send_message when using SQS?

### DISCUSSION THREAD

**Discussion by JAY-Chan9yu** [celery/celery#8084] [Q&A]
hello! i use SQS + Celery.

I want to get response of`sqs.send_message` when is use `send_task`.
https://boto3.amazonaws.com/v1/documentation/api/latest/guide/sqs-example-sending-receiving-msgs.html

### What i want
```python
app = Celery(broker="sqs://")
async_result: AsyncResult = app.send_task(event)
# pseudo code
sqs_message_id = async_result.sqs_result["MessageId"] <- i want to use it for log
logger.info(f"produce event - sqs_message_id : {sqs_message_id}")
```
#### kombu/transport/SQS.py 
This is debug code for check response of sqs.send_message.
<img width="464" alt="스크린샷 2023-02-23 오후 7 05 43" src="https://user-images.githubusercontent.com/24591259/220876761-39b372cc-b2ce-4a5e-9c29-ce78fc6c185f.png">

why `send_task` of Celery does not return `sqs.send_message`?
Isn't that a useful feature you think?😅

### **first,** kombu channel._put(sqs) does not return response.
<img width="400" alt="스크린샷 2023-02-23 오후 6 22 26" src="https://user-images.githubusercontent.com/24591259/220873322-14a8dce8-f126-42fd-8b17-3181e5e0a841.png">

###  **second,** Celery do not use response of `amqp.send_task_message` in `Celery.send_task`.
<img width="500" alt="스크린샷 2023-02-23 오후 6 23 10" src="https://user-images.githubusercontent.com/24591259/220873280-baa92c2b-eb3d-4ecc-ab73-5effa85b1578.png">

I hope there are people who think like me.😄
Thank you!

**Comment by auvipy** [comment]
I am open to review a PR for fixing the issue wiith unit tests

**Comment by blacklc** [comment]
I have the same question about this.

I use sqs to be celery broker. After send the task messages to broker, I can't see the message from the aws sqs console.

Maybe the messages was not sent to the sqs？To prove it, i want to check the api response from sqs api., but celery does not save the response.
